### ISSUE TITLE
Broken in Firefox 60

### ISSUE THREAD

**Issue by timmc** [deckgo/deckdeckgo-starter#45]
I visited the beta site in an up to date Firefox, version 60.9.0esr. (This browser version is less than a year old.) Pressing either the right arrow key or the next-slide button does nothing more than change the circle in the lower left from blue to white. I also get this in the browser console: `TypeError: Argument 1 of IntersectionObserver.observe is not an object.`

**Comment by peterpeterparker** [comment]
@timmc thx for the feedback. It aligns another feedback I had on Slack which I wasn't able to reproduce on Mac with Firefix at least.

At you facing that error on Linux or Windows maybe? 

With "beta site" do you mean, which URL exactly where u trying?

Thx in advance for the further info

**Comment by timmc** [comment]
Ah, I guess it's not the root page on that domain, is it. Here's the actual URL: https://beta.deckdeckgo.io/daviddalbusco/introducing-deckdeckgo/

I'm seeing this on Debian Linux.

**Comment by peterpeterparker** [comment]
Thx for the further info. Sounds like some standard might not yet be implemented by Firefox on Linux but I'll have to double check first.

Really appreciated, thx for all the details

**Comment by timmc** [comment]
It's doubtful it has anything to do with Debian's modifications to Firefox. Their packaging is very close to stock. But note that this is Firefox 60 (Extended Support Release), not Firefox 69, so you might be unknowingly relying on a feature only implemented in the past year.

**Comment by peterpeterparker** [comment]
@timmc I tried again with Firefox v60.9.0esr on my Mac and I finally do see the problem too.
I also see other visual error (like components being not correctly formatted).

Then I suspected, like I said above, a platform/browser compatibility error and therefore I checked [can I use](https://caniuse.com/#search=custom%20elements) about custom elements and noticed that Firefox before v63 didn't support them.

For such cases, Stencil components are shipped with a Polyfill and we even add the [@webcomponents/custom-elements](https://github.com/deckgo/deckdeckgo-starter/blob/master/package.json#L60) in the kit but seems that it isn't enough for Firefox v60.9.0esr.

I also double checked again that it does work correctly with Firefox v69.

Therefore I would say that there isn't much I could do unfortunately 😔

Let me know what you think about it.
